common/lib/capa/capa/capa_problem.py

```diff
@@ -857,6 +862,12 @@ def _preprocess_problem(self, tree):  # private
                 entry.attrib['id'] = "%s_%i_%i" % (self.problem_id, response_id, answer_id)
                 answer_id = answer_id + 1
 
+            # Find the label and save it for html transformation step
+            responsetype_label = response.find('label')
+            problem_data[self.problem_id + '_' + str(response_id)] = {
+                'label': responsetype_label.text if responsetype_label is not None else ''
+            }
```

Comment:
NOTE: This logic is being updated in https://github.com/edx/edx-platform/pull/13040